### ISSUE TITLE
How to generate dataset for a new environment

### ISSUE THREAD

**Issue by etri-jeon0123** [facebookresearch/home-robot#434]
We are constantly trying to use our environment in Habitat simulators, but it is not easy.

However, I'm working on the same process after reading your newly published paper 
(paper: Habitat Synthetic Scenes Dataset (HSSD-200)).

In other words, we also produced 2D drawings in 3D space using Floorplanner and extracted them as PNG files.

However, the problem now is that the data types extracted from Floorplanner are all PNG or JPG, and I wonder how you guys recreated the 3D space (.glb, .obj, etc). (When I asked the Floorplanner company, we received a reply that we could not extract the 3D data format (.glb, .obj, etc). )

Therefore, I wonder how you recreated the PNG file into a 3D space (.glb or .obj) and also how you did segmentation work for each object.

In other words, I wonder if your program is newly recreated or if it is a function provided by the Floorplanner program.

**Comment by jeonhuhuhu** [comment]
> Our scenes were originally authored as 3d scenes using the floorplanner tools, we didn't go from 2d to 3d images. Unfortunately we can't help you with this.

Now, we're making our environment as simple as you.

However, the HSSD dataset paper does not explain exactly how the scene data was created.

How do we work on and extract 3D scenes(.glb) from floorplanner tools.

I wonder if you extract it as a .glb file.

Probably, the questioner above also seems to be wondering how to extract 3D scenes(.glb) from the floor planner tool, rather than asking for conversion from 2D to 3D images. (If we don't pay, Floorplanner tool can only extract 2D png or jpg file format.)

Can I extract 3D scenes with .glb format data if I pay to buy an account?
